packages/filecoin-number/package.json

```diff
@@ -14,7 +14,8 @@
     "build-types": "../../node_modules/.bin/tsc --emitDeclarationOnly",
     "build": "npm run clean && npm run build-types && npm run build-node && npm run build-browser && npm run build-module",
     "prepublishOnly": "npm run build",
-    "test": "npm run build && jest",
+    "test-skip": "npm run build && ../../node_modules/.bin/jest",
```

Comment:
The README suggests a `APIADDR` var needs to be set?  We could set this as a secret and pass this in as variable.  Please advise on how to proceed here.

Reply:
Gah, that's stale. We can get rid of that. 

However, `filecoin-wallet-provider` could benefit form having a secret passed as an env var (just for testing). If it's easy, could you add the secret `apiAddress` and `token` (used [here](https://github.com/glifio/modules/pull/22/files#diff-2b01dbf04a098f3ee9dcfa5163056e971f37eedb9ef53b0d838965516347a222R17))

Reply:
Will follow up on this part.